collection_requirements.rst

```diff
@@ -267,11 +267,10 @@ allowed.
 These guidelines are the policy for inclusion in the Ansible package and are in addition to any
 licensing and legal concerns that may otherwise affect your code.
 
-
 Repository management
 =====================
 
-Every collection MUST have a public SCM repository, and releases of the collection MUST be tagged in this repository.
+Every Community collection MUST have a public SCM repository, and releases of the collection MUST be tagged in this repository.
```

Comment:
```suggestion
Every collection MUST have a public SCM repository, and releases of the collection MUST be tagged in this repository.
```
Same here. This applies to **all** collections included in Ansible, not only to `community.*` collections.

Reply:
`Community` is not about `community.*` collections. Its aim is to highlight that this is not about collections living on Automation Hub

Reply:
Solved